api/v1alpha4/awsidentity_types.go

```diff
@@ -70,7 +70,7 @@ type AWSRoleSpec struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:path=awsclusterstaticidentities,scope=Cluster,categories=cluster-api
+// +kubebuilder:resource:path=awsclusterstaticidentity,scope=Cluster,categories=cluster-api
```

Comment:
Is this change unintentional?

Reply:
this change is intentional @sedefsavas as i see it was giving some issues during manifest generation, after changing this it was fine, i cross-checked in other identities as well, its same

Reply:
// +kubebuilder:resource:path=`<plural name of your resources>`

It is plural if in other identities: 
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/c998877442736bdfcf3a4084db69ea162e0b3a05/api/v1alpha4/awsidentity_types.go#L149

Reply:
ok i think it got added unintentionally , i added the plural form before :) I will correct it